<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlg_extract</class>
 <widget class="QDialog" name="dlg_extract">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>163</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Extract from PDB</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Extract Elements</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="textLabel1">
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>32767</height>
           </size>
          </property>
          <property name="text">
           <string>Extract:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_pFillRange">
          <property name="text">
           <string>All</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QComboBox" name="m_pMethod">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <item>
           <property name="text">
            <string>Chain</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Model</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Frame</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Carbon Alpha</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Backbone</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Secondary Structure</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="m_pExtractRangeLE">
          <property name="toolTip">
           <string>Input a comma separated range to extract from, i.e 1-3,A-B,9,K</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="m_pExtract">
       <property name="text">
        <string>Extract</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_pCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>m_pExtract</tabstop>
  <tabstop>m_pMethod</tabstop>
  <tabstop>m_pExtractRangeLE</tabstop>
  <tabstop>m_pFillRange</tabstop>
  <tabstop>m_pCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
